packages/hint/docs/user-guide/development-flow-integration/circleci.md

```diff
@@ -0,0 +1,203 @@
+<!-- markdownlint-disable MD024 -->
+# Using CircleCI and webhint
+
+[CircleCI][] is a CI/CD service you can use to run webhint
+to test your website.
+
+## Local testing
```

Comment:
I find the term "Local" here a bit confusing since we're running in CI, not on a user's local machine.

@molant @hxlnt What do you think of using the header "Source code analysis" here instead of "Local testing" and "Live site analysis" later instead of "Online".

Reply:
I'm OK with that change. Can you please make a "suggestion" and I'll merge when/if @hxlnt gives the 👍?